x2gbfs/providers/moqo.py

```diff
@@ -295,3 +301,7 @@ def _pricing_plan_ids(self, vehicle_type: str) -> list[str]:
 
 class StadtwerkTauberfrankenProvider(MoqoProvider):
     pass
+
+
+class ZeagEnergieProvider(MoqoProvider):
+    pass
```

Comment:
As long as the superclass' default behavior is sufficient, you even would not need to define a subclass. For now, we could get rid of `StadtwerkTauberfrankenProvider`and `ZeagEnergieProvider` and just use `MoqoProvider`